README.md

```diff
@@ -309,6 +309,17 @@ if err != nil {
 }
 fmt.Print(status, status.ChannelId)
 ```
+## Note on usage of ablytest package
+Although `ablytest` package is available as a part of `ably`, we do not recommend using it as a sandbox for your own testing, since it's specifically intended for client library SDKs and we don’t provide any guarantees for support or that it will remain publicly accessible.
```

Comment:
```suggestion
Although the `ablytest` package is available as a part of ably-go, we do not recommend using it as a sandbox for your own testing, since it's specifically intended for client library SDKs and we don’t provide any guarantees for support or that it will remain publicly accessible.
```